dali/kernels/slice/slice_flip_normalize_permute_pad_common.h

```diff
@@ -45,9 +45,9 @@ struct SliceFlipNormalizePermutePadArgs {
   TensorShape<Dims> in_shape;
   std::array<bool, Dims> flip;
   std::array<int, Dims> permuted_dims;
-  SmallVector<float, 4> mean;
-  SmallVector<float, 4> inv_stddev;
-  SmallVector<float, 4> fill_values;
+  SmallVector<float, 8> mean;
```

Comment:
the args are now captured by the lambda. Increasing the static size to prevent allocations